Sources/MongoSwift/APM.swift

```diff
@@ -504,7 +504,7 @@ public enum MongoEventType {
     case serverMonitoring
 }
 
-/// An extension of MongoClient to add monitoring capability for commands and server discovery and monitoring.
+/// An extension of MongoClient to add monitoring capability for commands and server discovery and moni toring.
```

Comment:
typo?

Reply:
yeah, this is a draft anyway to inform a design document